var/spack/repos/builtin/packages/qwt/package.py

```diff
@@ -25,26 +25,22 @@
 from spack import *
 
 
-class Qwt(Package):
+class Qwt(QMakePackage):
     """The Qwt library contains GUI Components and utility classes which are
     primarily useful for programs with a technical background. Beside a
     framework for 2D plots it provides scales, sliders, dials, compasses,
     thermometers, wheels and knobs to control or display values, arrays, or
     ranges of type double.
     """
     homepage = "http://qwt.sourceforge.net/"
-    url      = "https://downloads.sourceforge.net/project/qwt/qwt/5.2.2/qwt-5.2.2.tar.bz2"
+    url      = "https://sourceforge.net/projects/qwt/files/qwt/6.1.3/qwt-6.1.3.tar.bz2"
 
+    version('6.1.3', '19d1f5fa5e22054d22ee3accc37c54ba')
     version('5.2.2', '70d77e4008a6cc86763737f0f24726ca')
 
-    depends_on("qt")
-
-    def install(self, spec, prefix):
+    depends_on('qt+opengl')
 
+    def patch(self):
         # Subvert hardcoded prefix
-        filter_file(r'/usr/local/qwt-\$\$VERSION', prefix, 'qwtconfig.pri')
-
-        qmake = which('qmake')
-        qmake()
-        make()
-        make("install")
+        filter_file('/usr/local/qwt-$$VERSION', self.prefix, 'qwtconfig.pri',
```

Comment:
In version 6.1.3, the string to replace is '/usr/local/qwt-$$QWT_VERSION'. Can you make this a regex to catch that too?